src/chef-server-ctl/Gemfile.lock

```diff
@@ -46,100 +47,122 @@ GEM
   specs:
     addressable (2.6.0)
       public_suffix (>= 2.0.2, < 4.0)
-    appbundler (0.12.0)
+    appbundler (0.13.0)
```

Comment:
Looks like the current build failures are because 0.13.0 got yanked from Rubygems. @lamont-granquist Do you know if this is expected?